packages/vuetify/dev/Playground.template.vue

```diff
@@ -1,18 +1,286 @@
 <template>
```

Comment:
Docs examples should cover most of it
I think it is better to exclude this file from this PR